controllers/common.go

```diff
@@ -122,51 +110,25 @@ func (c commonController) Authenticate(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	flowKey := string(uuid.NewUUID())
-
-	flows := map[string]string{}
-
-	if err := session.GetObject("flows", &flows); err != nil {
-		logErrorAndWriteResponse(w, http.StatusInternalServerError, "failed to decode session data", err)
-		return
-	}
-
-	flows[flowKey] = token
-
-	if err := session.PutObject(w, "flows", flows); err != nil {
-		logErrorAndWriteResponse(w, http.StatusInternalServerError, "failed to encode session data", err)
-		return
-	}
-
 	keyedState := exchangeState{
 		AnonymousOAuthState: state,
-		Key:                 flowKey,
```

Comment:
Since we're removing the ability to handle multiple oauth flows in a single session with this (which is ok by me), let's also remove the `key` from the `exchangeState`..